recipe/activate-cross-python.sh

```diff
@@ -20,6 +20,11 @@ if [[ "${CONDA_BUILD:-0}" == "1" && "${CONDA_BUILD_STATE}" != "TEST" ]]; then
         --cc ${CC} \
         --cxx ${CXX:-c++}
 
+    # Undo cross-python's changes
+    # See https://github.com/conda-forge/h5py-feedstock/pull/104
+    rm $BUILD_PREFIX/venv/$(basename $sysconfigdata_fn)
+    cp $sysconfigdata_fn $BUILD_PREFIX/venv/lib/$(basename $sysconfigdata_fn)
+
```

Comment:
I think this needs to check that the file exists before removing it. At least that seems to be the error in th [h5py-feedstock build failure](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=413005&view=logs&j=abc074f0-60c0-536a-336f-04e4d67bf687&t=cc3249ce-60a3-5d53-93c3-b1bfc52f0e78&l=258) xref https://github.com/conda-forge/h5py-feedstock/pull/104#issuecomment-970647822